src/ray/core_worker/core_worker.cc

```diff
@@ -2283,10 +2283,17 @@ void CoreWorker::PlasmaCallback(SetResultCallback success,
                                 std::shared_ptr<RayObject> ray_object, ObjectID object_id,
                                 void *py_future) {
   std::vector<std::shared_ptr<RayObject>> vec;
+  bool object_is_local = false;
+
   // Check if object is available before subscribing to plasma.
-  if (Get(std::vector<ObjectID>{object_id}, 0, &vec).ok() && vec.size() > 0) {
+  if (Contains(object_id, &object_is_local).ok() && object_is_local) {
```

Comment:
Changed it to `Contains` so it won't trigger unnecessary pull requests and immediately cancel them.

Reply:
Could you throw a comment in explaining this behavior?